### PR TITLE
chore: Update Readme.md to reflect latest changes

### PR DESCRIPTION
**Pull request by philasmar**
*Issue #, if available:*

*Description of changes:*
* Update Readme.md to reflect latest changes about supporting existing beanstalk deployments. Since the tool is no longer specifically tied to the CDK, I changed the wording to reflect that.
* Added a link to the dev guide.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by normj — APPROVED** [review]

**Review by 96malhar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -93,13 +94,23 @@ Choose deployment option (recommended default: 1)
...
+
+The deploy tool will now detect existing Elastic Beanstalk environments in your AWS account and list them alongside the existing CloudFormation stacks (that were created using the deploy tool). You will be able to select an existing Beanstalk environment and deploy your application that environment as well as updating some settings that were available in the old AWS Visual Studio toolkit experience. On the backend, the deploy tool is using the AWS .NET SDK to perform the deployment to existing Beanstalk Environments.
+
+Note: Deploying to existing Beanstalk Environments will not create a new CloudFormation stack and migrate the existing resources to be managed by CloudFormation. In order to do that, you will need to deploy your application to a new stack which will use the AWS CDK to manage all the resource in CloudFormation.
```
Comment (96malhar) [inline README.md:107]: _"and **will not** migrate the existing resource...."_

**Review by 96malhar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (philasmar) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # AWS .NET deployment tool
 
 
-This repository contains the AWS .NET deployment tool for .NET CLI - the opinionated tooling that simplifies deployment of .NET applications with minimum AWS knowledge. The tool suggests the right AWS compute service to deploy your application to.  It then builds and packages your application as required by the chosen compute service, generates the deployment infrastructure, deploys your application by using the Cloud Development Kit (CDK), and displays the endpoint.
+This repository contains the AWS .NET deployment tool for .NET CLI - the opinionated tooling that simplifies deployment of .NET applications with minimum AWS knowledge. The tool suggests the right AWS compute service to deploy your application to.  It then builds and packages your application as required by the chosen compute service, generates the deployment infrastructure, deploys your application by using the appropriate deployment engine (Cloud Development Kit (CDK) or native service APIs), and displays the endpoint.
```
[the remark this answers is not in the document]
Comment (philasmar) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -93,13 +94,23 @@ Choose deployment option (recommended default: 1)
...
+For users coming from the previous AWS Visual Studio Toolkit experience who have previous Elastic Beanstalk deployments and environments, you will be able to use the deploy tool to deploy to those environments.
+
+#### Elastic Beanstalk backwards compatibility support
+Currently, the AWS Visual Studio toolkit allows users to deploy their ASP.NET Core web applications to Elastic Beanstalk using a wizard that uses the AWS .NET SDK on the backend to perform the deployment. This process creates the necessary Elastic Beanstalk resources such as a Beanstalk Application and Beanstalk Environment using the AWS .NET SDK. 
+
+The new deploy tool uses AWS CDK to perform the deployments to AWS which create a CloudFormation stack that manages all the Elastic Beanstalk resources. This means that the deploy tool cannot update the existing Beanstalk resources that were created using the old AWS Visual Studio toolkit experience. In order to tackle this issue, the deploy tool now supports deploying to existing Elastic Beanstalk environments, in addition to CloudFormation stacks.
```
[the remark this answers is not in the document]
Comment (philasmar) [reply README.md]: Done

File: README.md [hunk 4]
```diff
@@ -93,13 +94,23 @@ Choose deployment option (recommended default: 1)
...
+
+The new deploy tool uses AWS CDK to perform the deployments to AWS which create a CloudFormation stack that manages all the Elastic Beanstalk resources. This means that the deploy tool cannot update the existing Beanstalk resources that were created using the old AWS Visual Studio toolkit experience. In order to tackle this issue, the deploy tool now supports deploying to existing Elastic Beanstalk environments, in addition to CloudFormation stacks. 
+
+The deploy tool will now detect existing Elastic Beanstalk environments in your AWS account and list them alongside the existing CloudFormation stacks (that were created using the deploy tool). You will be able to select an existing Beanstalk environment and deploy your application that environment as well as updating some settings that were available in the old AWS Visual Studio toolkit experience. On the backend, the deploy tool is using the AWS .NET SDK to perform the deployment to existing Beanstalk Environments.
```
[the remark this answers is not in the document]
Comment (philasmar) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -93,13 +94,23 @@ Choose deployment option (recommended default: 1)
...
+
+Note: Deploying to existing Beanstalk Environments will not create a new CloudFormation stack and will not migrate the existing resources to be managed by CloudFormation. In order to do that, you will need to deploy your application to a new stack which will use the AWS CDK to manage all the resource in CloudFormation.
```
[the remark this answers is not in the document]
Comment (philasmar) [reply README.md]: Done

File: README.md [hunk 6]
```diff
@@ -93,13 +94,23 @@ Choose deployment option (recommended default: 1)
...
 
 ### Blazor WebAssembly applications
 Blazor WebAssembly applications can be deployed by creating and configuring an Amazon S3 bucket for web hosting. Your Blazor application will then be uploaded to the S3 bucket.
```
[the remark this answers is not in the document]
Comment (philasmar) [reply README.md]: Done

File: README.md [hunk 7]
```diff
@@ -113,6 +124,7 @@ We welcome community contributions and pull requests. See [CONTRIBUTING](https:/
 
 ## Additional Resources
```
[the remark this answers is not in the document]
Comment (philasmar) [reply README.md:126]: Done

**Review by ik130 — APPROVED** [review]

**Pull request merged**